interface-elements-for-web/articles/grid/filtering/filtering-grid-data.md

```diff
@@ -40,14 +40,17 @@ To remove the column filter, clear the text in the auto-filter row. To remove th
 
 ![ASPxGridView_ClearAutoFilterRow](../../../images/img7158.png)
 
-&nbsp;
+
 
 **Search Panel**
 
-To filter data and highlight search results, use the [Search Panel](search-panel.md).
+Use the [Search Panel](search-panel.md) to filter data and highlight search results.
+
+
+**Filter Builder**
 
-&nbsp;
+Use the [Filter Builder](creating-complex-filter-criteria-with-the-filter-control.md) to create complex filter criteria.
 
-**Filter Control**
+**Customization Dialog**
 
-To create complex filter criteria, use the [Filter Control](creating-complex-filter-criteria-with-the-filter-control.md).
\ No newline at end of file
+Use the [customization dialog](../customization-dialog/filtering-page.md) to filter grid data by entering filter criteria.
```

Comment:
You can specify the filter criteria in the [customization dialog](../customization-dialog/filtering-page.md) to filter grid data?